albert_garcia/fun_ruby/my_enumerable.rb

```diff
@@ -0,0 +1,12 @@
+module MyEnumerable
+    def my_each
+        for i in self
+            puts i * 10
+        end
+    end
+  end
+  class Array
+     include MyEnumerable
+  end
+  [1,2,3,4].my_each { |i| puts i } 
+  [1,2,3,4].each { |i| puts i * 10 } 
```

Comment:
[1,2,3,4].my_each { |i| puts i * 10 } //this worked because the built in each was called